versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.3.0"
```

Comment:
Not necessary for the root module. It doesn't use optionals.

Reply:
noted and updated!